connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/DistributedHerder.java

```diff
@@ -1063,6 +1076,132 @@ public int generation() {
         return generation;
     }
 
+    @Override
+    public void restartConnectorAndTasks(
+            RestartRequest request,
+            Callback<ConnectorStateInfo> callback
+    ) {
```

Comment:
nit: similar style as elsewhere in this file
```suggestion
    public void restartConnectorAndTasks(RestartRequest request, Callback<ConnectorStateInfo> callback) {
```